src/components/views/settings/UserProfileSettings.tsx

```diff
@@ -76,6 +80,25 @@ const ManageAccountButton: React.FC<ManageAccountButtonProps> = ({ externalAccou
     </AccessibleButton>
 );
 
+const SignOutButton: React.FC = () => {
+    const client = useMatrixClientContext();
+
+    const onClick = useCallback(async () => {
+        if (await shouldShowLogoutDialog(client)) {
+            Modal.createDialog(LogoutDialog);
+        } else {
+            defaultDispatcher.dispatch({ action: "logout" });
+        }
+    }, [client]);
+
+    return (
+        <AccessibleButton onClick={onClick} element="a" kind="danger_outline">
```

Comment:
Why is this an anchor vs a button/div?

Reply:
ah, it doesn't - I probably ust copied it from the other button.